### PR TITLE
rules_shellcheck: add 0.2.3

### PR DESCRIPTION
**Pull request by aignas**

### CODE REVIEW

File: modules/rules_shellcheck/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "email": "240938+aignas@users.noreply.github.com",
+      "github": "aignas",
+      "name": "Ignas Anikevicius"
+    }
+  ],
+  "repository": [
+    "github:aignas/rules_shellcheck"
+  ],
+  "versions": [
+    "0.2.3",
```
Comment (fmeum) [inline modules/rules_shellcheck/metadata.json:14]:
```suggestion
    "0.2.3"
```

**Comment by fmeum** [comment]
@aignas I fixed the Json, but the module file differs from the one in the archive.

**Comment by alexeagle** [comment]
FYI I just added shellcheck in https://github.com/aspect-build/rules_lint/pull/32
Happy to see both projects thrive, but I'd also be happy to combine efforts.

**Comment by aignas** [comment]
@fmeum, thanks for looking into this. I created this manually, because I can't seem to get the BCR publishing app working, so I'll look into how to get that working.

@alexeagle, thanks for letting me know. I'll use `rules_shellcheck` for experimenting with bazel concepts for now but will keep in mind to point users to `rules_lint` if I decide to abandon the project.

**Pull request closed without merging**